### PR TITLE
-Adding support for multiple orientations

### PR DESCRIPTION
**Pull request by Strobocop**
Thanks for the great library! I'm using this in an iPad app that only supports landscape and needed to make a few changes to support the orientation. Seems to work just fine in other orientations.

### CODE REVIEW

**Comment by zenangst** [comment]
I like this; what do you think @vadymmarkov ?
The only thing that I see is that you are using 4 spaces and not 2 spaces like we do, mind fixing that @Strobocop

File: Sources/BarcodeScannerController.swift [hunk 1]
```diff
@@ -449,3 +449,26 @@ extension BarcodeScannerController: HeaderViewDelegate {
     dismissalDelegate?.barcodeScannerDidDismiss(self)
   }
 }
+
+extension AVCaptureVideoOrientation {
```
Comment (vadymmarkov) [inline Sources/BarcodeScannerController.swift]: I would move it to a separate file, there are a lot of code in the controller already.

**Comment by tapwork** [comment]
ups...didn't see that there is already a PR for the orientations:
I added another one: https://github.com/hyperoslo/BarcodeScanner/pull/31

**Comment by onmyway133** [comment]
Since I just merged https://github.com/hyperoslo/BarcodeScanner/pull/31, I think we can close this one. Thanks @Strobocop

**Pull request closed without merging**